GitHubMilestones.ps1

```diff
@@ -164,12 +184,12 @@ function New-GitHubMilestone
         The title of the milestone.
 
     .PARAMETER State
-        Only milestones with this state are returned, either open or closed. Default: open
+        The state of the milestone, either Open or Closed.
```

Comment:
Similar to not needing to say what the default is, you also don't need to indicate the possible options....PowerShell will do that for you when generating documentation if there's a `[ValidateSet()]` on a parameter.